libs/net/dal/Configuration/SeriesConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public override void Configure(EntityTypeBuilder<Series> builder)
         builder.Property(m => m.SourceId).IsRequired(false);
         builder.Property(m => m.AutoTranscribe).IsRequired();
         builder.Property(m => m.UseInTopics).IsRequired();
+        builder.Property(m => m.IsCBRASource).IsRequired(false);
```

Comment:
According to your entity it is required.

Reply:
I will update. thanks

Reply:
updated. thanks